src/main/java/pl/agh/edu/master_diet/core/model/database/Achievement.java

```diff
@@ -1,9 +1,17 @@
 package pl.agh.edu.master_diet.core.model.database;
 
-import lombok.*;
-import org.hibernate.annotations.Type;
-
-import javax.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Column;
```

Comment:
About all these imports - maybe let's use import * ? You and @RafisSomeone keep replacing yourself with these imports :smile:

Reply:
What about IntelliJ import organization? We would have to disable this feature in commit dialog.

Reply:
Yeah, so you can choose there an option to replace for import *

Reply:
Ok, replaced with `import *`